plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -65,7 +65,7 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
                 binder -> binder.bind(OpenTelemetry.class).toInstance(context.getOpenTelemetry()),
                 binder -> binder.bind(CatalogName.class).toInstance(new CatalogName(catalogName)),
                 new JdbcModule(),
-                module);
+                requireNonNull(moduleSupplier.get(), "moduleSupplier.get() is null"));
```

Comment:
module.get()

Bootstrap wraps module with ImmutableList.copyOf() which does the null check